proto/markdown.tmpl

```diff
@@ -0,0 +1,107 @@
+<!-- This file is automatically generated. DO NOT EDIT! -->
+<!-- Needed because of https://github.com/pseudomuto/protoc-gen-doc/issues/274 -->
```

Comment:
I added a reference to the issue, because then our [github action](https://github.com/ory/closed-reference-notifier/) will notify us when it is fixed upstream :wink: